packages/components/src/ui/styles/theme/utils.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import { get } from '../../create-styles';
+import colorize from 'tinycolor2';
```

Comment:
Just syncing up with @ajlende here, as I think tinycolor is also used for some of the Duotone stuff.